training_event/urls.py

```diff
@@ -10,6 +10,9 @@
 router.register(r'off-campus-events',
                 training_event.views.OffCampusEventViewSet)
 router.register(r'enrollments', training_event.views.EnrollmentViewSet)
+router.register(r'role-choices',
```

Comment:
You can make it an extra action of your `Record`'s viewset, I think this API is for your off-campus record only, no need to add a viewset. (Although the best place might be `EventCoefficient`, but we have no viewset for it).